@xen-orchestra/lite/src/components/Table.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import styled from 'styled-components'
+import { withState } from 'reaclette'
+
+import IntlMessage from './IntlMessage'
+
+export type Column<Type> = {
+  header: React.ReactNode
+  id?: string | number
+  render: { (item: Type): string | JSX.Element }
+}
+
+export type Item<T> = T & {
+  id?: string
+}
```

Comment:
Since `T` will always be `unknown`, I'm wondering if this is necessary. Ideally, the type should come from the outside but I don't know if it's possible in this case.